types/spectest/tests/consensusdata/invalid_sync_committee_cntribution_validation.go

```diff
@@ -1,6 +1,16 @@
 package consensusdata
 
+import "github.com/bloxapp/ssv-spec/types/testingutils"
+
 // InvalidSyncCommitteeContributionValidation tests an invalid consensus data with sync committee contrib.
-func InvalidSyncCommitteeContributionValidation() *SpecTest {
-	panic("implement")
+func InvalidSyncCommitteeContributionValidation() *ConsensusDataTest {
+
+	cd := testingutils.TestSyncCommitteeContributionConsensusData
+	cd.DataSSZ = testingutils.TestingAttestationDataBytes
+
+	return &ConsensusDataTest{
+		Name:          "invalid sync committee contribution",
+		ConsensusData: *cd,
+		ExpectedError: "could not unmarshal ssz: four",
```

Comment:
you know what "four" means here?
it comes from ssz lib

Reply:
seems to be like an error code or something, that's pretty funny, from the fastssz lib:
```go
		if offset > endOffset {
			return fmt.Errorf("four")
		}
		if endOffset > size {
			return fmt.Errorf("five")
		}
```